src/main/java/com/fruitshop/db/DataBase.java

```diff
@@ -0,0 +1,13 @@
+package com.fruitshop.db;
+
+import com.fruitshop.model.Fruit;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class DataBase {
+    private static Map<String, Fruit> fruitsInShop = new TreeMap<>();
+
+    public static Map<String, Fruit> currentDataBase() {
+        return fruitsInShop;
+    }
```

Comment:
Seems like this method is redundant. You can just make your map final and nothing will change